src/components/layout/app-menu.js

```diff
@@ -1,56 +1,92 @@
 import React from 'react'
 import styled from 'styled-components/macro'
-import PropTypes from 'prop-types'
-import { Menu, Icon } from 'antd'
 import { NavLink } from 'react-router-dom'
+import { Menu, Dropdown, Button, Icon } from 'antd'
+import MenuIcon from 'assets/images/menu-icon.png'
 
-const StyledMenu = styled(Menu)`
+const DesktopMenu = styled(Menu)`
+  @media (max-width: 768px) {
```

Comment:
I would prefer to choose another breakpoint (900px maybe) as [this](https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862) suggests) but I saw that through the codebase there are already references to the `768px`, and also other values, so let's leave it like this for now, but keep in the backlog a new PR to standarize them.

Reply:
yep I prefer that too. I can do that since I have nothing else for now to do